app/src/main/java/com/vitorpamplona/amethyst/ui/navigation/DrawerContent.kt

```diff
@@ -206,66 +214,84 @@ fun ListContent(
     navController: NavHostController,
     scaffoldState: ScaffoldState,
     modifier: Modifier,
-    accountViewModel: AccountStateViewModel
+    accountViewModel: AccountStateViewModel,
+    account: Account,
 ) {
     val coroutineScope = rememberCoroutineScope()
+    var backupDialogOpen by remember { mutableStateOf(false) }
 
-    Column(modifier = modifier) {
-        LazyColumn() {
```

Comment:
I'm not sure why this was previously `Column() { LazyColumn() { Column() }}}` It seems to work fine with just a single `Column`. Can revert if I'm missing something...